packages/web-react/src/types/collapse.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactNode } from 'react';
+import { ChildrenProps, ClickEvent, StyleProps, Booleanish } from './shared';
+
+export type CollapseElementType = 'div' | 'article' | 'section' | 'main' | 'header' | 'footer' | 'span';
+
+export type CollapseResponsiveType = undefined | 'mobile' | 'tablet' | 'desktop';
+
+export type CollapseTriggerProps = {
+  onClick: (event: ClickEvent) => void;
+  className?: string | undefined;
+  'aria-expanded': Booleanish | undefined;
```

Comment:
Is this ok? I think that `undefined` => `false`. So we should handle somewhere the right value.

Reply:
Good point, i set it to `Booleanish` only, so it includes boolean already